packages/shared/lib/core/network/helpers/buildClientOptions.ts

```diff
@@ -0,0 +1,8 @@
+import { ClientOptions } from '@iota/wallet'
+import { INetwork, INode } from '../interfaces'
+
+export function buildClientOptions(network: INetwork, nodes: INode[]): ClientOptions {
```

Comment:
`network` is not needed anymore. I'm also not sure whether this function is still required, given that we already have a getClientOptions and there is no building happening. I think this might also simplify `getDafaultClientOptions`.

Reply:
ahh yes, I was thinking that we might need this when we start updating the client options through the settings etc.